graceoncampus/goc-web-v3
# Add pickup locations popup

Testing Steps
------------------------------------------------------------------------
1. yarn && yarn build && yarn start
2. navigate to localhost:3000
3. ensure that pickup locations renders onHover for desktop
4. switch to mobile viewport (user agent)
5. ensure that pickup locations render onClick and you can click outside
to get rid of it

Resources
------------------------------------------------------------------------
- Tippy JS (to create tooltip): https://atomiks.github.io/tippyjs/
- Figma Mock: https://www.figma.com/file/lW8XjLzvK1fRqzetVqOs7q/GOC?node-id=298%3A1247
